File: Suiki/app/src/main/java/com/suiki/suiki/Model/BddModel/Personne.java
package com.suiki.suiki.Model.BddModel;

import com.google.gson.annotations.SerializedName;

import org.simpleframework.xml.Element;

/**
 * Created by Massil on 11/01/2017.
 */

public class Personne {
    @SerializedName("_id")
    public String id;
    @SerializedName("nom")
    public String nom;
    @SerializedName("prenom")
    public String prenom;
    @SerializedName("email")
    public String email;
    @SerializedName("estPatient")
    public static boolean estPatient = true;
}

File: Service/model/suikiBdd/personne.js
/**
 * Created by Massil on 25/01/2017.
 */
var db = require('./../../config/db');

var Schema = db.Schema;

var personne = Schema({
    nom : 'String',
    prenom : 'String' ,
    email : 'String'
});

module.exports = db.model('Personne' , personne);


